apps/machine/zynq7/rsc_table.c

```diff
@@ -29,8 +29,6 @@
 /* Resource table entries */
 #define NUM_VRINGS                  0x02
 #define VRING_ALIGN                 0x1000
```

Comment:
You can move VRING_ALINE and NUM_VRINGS as well.

Reply:
done in the next round of commits